css-backgrounds-4/Overview.bs

```diff
@@ -185,26 +185,32 @@ Painting Area: the 'background-clip' property</h3>
 
 	<pre class=propdef>
 	Name: border-top-color, border-right-color, border-bottom-color, border-left-color
-	Value: <<color>>#
+	Value: <<color>> || <<1d-image>>
 	Initial: currentcolor
 	Applies to: all elements
 	Inherited: no
 	Percentages: n/a
-	Computed Value: the computed color
+	Computed Value: the computed color and/or a one-dimensional image function
 	Animation type: see prose
 	</pre>
 
 	<pre class="propdef shorthand">
 	Name: border-color
-	Value: <<color>>#{1,4}
+	Value: [ <<color>> || <<1d-image>> ]{1,4}
 	</pre>
 
 	These properties set the foreground color of the border specified
 	by the 'border-style' properties.
-	If a list of values is provided, the border is split into equal width bands of each color
-	along the direction of the side the border is applied on
-	(i.e. split horizontally on left and right borders and vertically on top and bottom borders),
-	starting outwards.
+
+	The stripes defined by <<1d-image>> follow the direction of
+	the side the border is applied on (i.e. horizontally on left and
+	right borders and vertically on top and bottom borders), starting outwards.
+	With that, the corresponding 'border-*-width' property defines the total
+	width of the stripes.
+
+	The stripes are painted above a given <<color>>. This allows the <<color>>
+	value to serve as a fallback, so any transparent stripes let the color
+	shine through.
 
 	When interpolating between borders with the same number of colors,
 	interpolation is performed individually per color band as <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>.
```

Comment:
Interpolation should probably be defined in the definition of the `stripes()` function, rather than here. And it should match what we do for gradients.

Reply:
I've moved it to the "Interpolation" section in CSS Images 4 and tried to align its definition with the one for gradients. Please have a detailed look at it!

Sebastian